python/tirex_tracker/__init__.py

```diff
@@ -873,9 +873,10 @@ def _export_ir_metadata(self, result: "Pointer[_Result]") -> None:
         ir_metadata["implementation"]["python"]["interactive"] = loads(
             self._python_info[Measure.PYTHON_IS_INTERACTIVE].value
         )
-        ir_metadata["implementation"]["script"]["path"] = loads(
-            self._python_info[Measure.PYTHON_SCRIPT_FILE_PATH].value
-        )
+        if Measure.PYTHON_SCRIPT_FILE_PATH in self._python_info:
+            ir_metadata["implementation"]["script"]["path"] = loads(
+                self._python_info[Measure.PYTHON_SCRIPT_FILE_PATH].value
+            )
```

Comment:
Nice check!

Reply:
This check is then also no longer necessary when the `PYTHON_SCRIPT_FILE_PATH` and `PYTHON_NOTEBOOK_FILE_PATH` are removed.